app/src/main/AndroidManifest.xml

```diff
@@ -50,6 +51,21 @@
 
         </activity>
 
+        <activity
```

Comment:
yes and same here
if we will install this app there will be 2 icons for run app and sample
as far as i see